modules/update-best-practices.adoc

```diff
@@ -36,6 +36,16 @@ In most cases this alert does not block patch updates, but you cannot perform a
 
 For more information about the `Upgradeable` condition, see "Understanding cluster Operator condition types" in the additional resources section.
 
+[id="sdn-support-removal"]
+== SDN support removal
```

Comment:
could be a `===` to nest it under `Ensure that the cluster is in an Upgradable state`?  Because this is one condition of many that could lead to the 4.16 cluster being `Upgradeble=False` which would need sorting before the cluster could update to 4.17.

Reply:
Absolutely! Makes sense to me :)